### DISCUSSION TITLE
Is pockethost down?

### DISCUSSION THREAD

**Discussion by tekojtrapkomo** [pockethost/pockethost#182] [Q&A]
Hi ppl. Is pockethost down or is it just me?

**Comment by benallfree** [comment]
Seems to be working. What are you seeing?

**Reply by tekojtrapkomo** [reply]
it was just down for 30 mins that day

**Comment by Mellbin** [comment]
@benallfree Seeing intermittent issues today, down for 20min then up for like 2min, has been going on for 2 hours now.

**Reply by benallfree** [reply]
Looking into it. Give me a few hours.
